packages/tools/generator-widget/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mendix/generator-widget",
-  "version": "8.14.0",
+  "version": "8.18.0",
```

Comment:
Is this large jump intended?

Reply:
Yes, we always align with the latest mx minor release.